addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -2037,7 +2037,7 @@
 		
 		"58"	//Jarate
 		{
-			"desp"			"Jarate: {neutral}Replaces jarate effect into a rage drain, removing equivalent of 400 damage"
+			"desp"			"Jarate: {neutral}Replaces jarate effect with a rage drain, drains 400 damage worth of rage"
```

Comment:
`Replaces jarate effect with 400 damage worth of rage drain`? Not sure what else it can be worded

Reply:
I'll just leave this one as is, it's not too bad